corgidrp/mocks.py

```diff
@@ -202,3 +206,143 @@ def create_default_headers(obstype="SCI"):
     exthdr['MISSING'] = False
 
     return prihdr, exthdr
+
+def create_astrom_data(filedir=None, field_path='~/corgidrp/tests/test_data/JWST_CALFIELD2020.csv'):
```

Comment:
we shouldn't have a path that's hard coded that depends on the user's directory. I think the best thing to do is to require it as an input argument (instead of an optional keyword). Then, your test code can reconstruct the file path.

Reply:
You can use python path manipulation functions in the test script to navigate to the corgidrp/tests/test_data folder. e.g.,:
```
# reconstruct where the dark is saved
calibdir = os.path.join(os.path.dirname(__file__), "testcalib")
dark_filename = "sim_dark_calib.fits"
```